doc/source/parquet.md

```diff
@@ -8,7 +8,7 @@ for more on Apache Parquet.
 
 ## Partitioning
 
-As outlined in [Spatial partitioning in Dask-GeoPandas](guide/spatial-partitioning.md), dask-geopandas can spatially partition datasets. These partitions are
+As outlined in [Spatial partitioning in Dask-GeoPandas](https://dask-geopandas.readthedocs.io/en/stable/guide/spatial-partitioning.html), dask-geopandas can spatially partition datasets. These partitions are
```

Comment:
```suggestion
As outlined in [Spatial partitioning in Dask-GeoPandas](guide/spatial-partitioning.ipynb), dask-geopandas can spatially partition datasets. These partitions are
```

This link needs to be relative so it always leads to appropriate version of the documentation, not to stable only.